### PR TITLE
refactor: cleanup

### PR DESCRIPTION
**Pull request by florian-hoenicke**
remove redundant code
some fixes

image

### CODE REVIEW

File: src/apis/gpt.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def chat(self, prompt: str):
         self.messages.append(chat_message)
```
Comment (joschkabraun) [inline src/apis/gpt.py:82]: Good change! We also need to set `verbose=True` in `ChatOpenAI`

**Review by joschkabraun — APPROVED** [review]
small suggestions, otherwise LGTM

File: src/cli.py [hunk 2]
```diff
@@ -67,8 +67,8 @@ def generate(
     generator = Generator(description, test, model=model)
     with get_openai_callback() as cb:
```
[the remark this answers is not in the document]
Comment (florian-hoenicke) [reply src/cli.py]: done

File: src/options/generate/generator.py [hunk 3]
```diff
@@ -52,6 +52,26 @@ def files_to_string(self, file_name_to_content, restrict_keys=None):
                 all_microservice_files_string += f'**{file_name}**\n```{tag}\n{file_name_to_content[file_name]}\n```\n\n'
         return all_microservice_files_string.strip()
 
+
+    def generate_and_persist_file(self, section_title, template, destination_folder, file_name, **kwargs):
```
[the remark this answers is not in the document]
Comment (florian-hoenicke) [reply src/options/generate/generator.py]: done

**Pull request merged**